date/index.js

```diff
@@ -171,37 +192,37 @@ const formatMap = {
 /**
  * Adds a locale to moment, using the format supplied by `wp_localize_script()`.
  *
- * @param {Object} settings Settings, including locale data.
+ * @param {Object} s Settings, including locale data.
  */
-function setupLocale( settings ) {
+function setupLocale( s ) {
```

Comment:
I believe this because of the ESlint rule we have, can't we find a name that's not a single letter? `localeSettings` or something.

Reply:
I love single letter variables, <a href="https://github.com/golang/go/wiki/CodeReviewComments#variable-names">blame Golang</a>. I'll switch it out